lib/showoff.rb

```diff
@@ -1375,13 +1375,16 @@ def valid_cookie
   end
 
   get %r{(?:image|file)/(.*)} do
-    path = params[:captures].first
-    full_path = File.join(settings.pres_dir, path)
-    if File.exist?(full_path)
-        send_file full_path
-    else
-        raise Sinatra::NotFound
-    end
+	path = params[:captures].first
+	full_path = File.join(settings.pres_dir, path)
+
+	if File.exist?(full_path)
+		send_file full_path
+	elsif File.exist?(path) # windows compatibility
```

Comment:
Right, but `send_file` expects a local path on the filesystem, so the fully qualified name is correct. I've validated on a windows VM and it works properly.

I wonder if you're running into an oddity about the virtual filesystem representation that showoff uses. I'm not terribly happy with it and I'd like to remove it, but I've got to be careful with its deprecation because it would break backwards compatibility pretty badly. There is an additional component in the path.

For example, if you have `foo.jpg` in the root of your presentation:

* In markdown, you'd use `![Foo](foo.jpg)`
* In HTML, you'd use `<img src="file/foo.jpg" />`

The paths are also relative from the slide, so if you have a slide of `intro/about.md`, then you'd need to use `![Foo](../foo.jpg)` in that slide.

Please reopen and/or file a ticket if you find something else awry. Thanks!